src/NetscriptFunctions/Singularity.ts

```diff
@@ -134,18 +134,20 @@ export function NetscriptSingularity(player: IPlayer, workerScript: WorkerScript
         return aug.prereqs.slice();
       },
     getAugmentationPrice: (_ctx: NetscriptContext) =>
-      function (_augName: unknown): number {
+      function (_augName: unknown, _returnBasePrice: unknown = false): number {
```

Comment:
yeah i still dont think we need the base price on this function imo, like will it ever be useful?

or is there a usecase im missing?